skbio/tree/_tree.py

```diff
@@ -2705,6 +2705,47 @@ def compare_tip_distances(self, other, sample=None, dist_f=distance_from_r,
         return dist_f(self_matrix, other_matrix)
 
     @experimental(as_of="0.4.0")
+    def bifurcate(self, insert_length=None, remove_singles=True):
+        r""" Restructures tree into a bifurcating tree
+
+        All nodes that have more than 2 children will
+        have additional intermediate nodes injected to ensure that
+        every node has only 2 children
```

Comment:
:bomb:

Now, this can be done with the current functionality in the PR.
It would be something like this
```
tree.children[0].bifurcate()
tree.children[1].bifurcate()
tree.children[2].bifurcate()
```
I admit, this is hack and not elegant.  What would be a proper alternative to handle this scenario?  Would it be helpful to add an additional parameter (i.e. `unrooted=True`)?